src/user/models/auth/auth.service.ts

```diff
@@ -89,12 +95,12 @@ export class AuthService {
       .getRawOne<User>();
   }
 
-  private verifySignature(address: string, signature: string): boolean {
-    const signatureMessage = this.getSignMessage(address);
+  private verifySignature(address: string, signature: string, companyLogin = false): boolean {
```

Comment:
For company login we should use the challenge-response method. The wallet signing message can be removed.